Library/Formula/ppss.rb

```diff
@@ -1,7 +1,7 @@
 require 'formula'
 
 class Ppss < Formula
-  homepage 'http://ppss.googlecode.com'
+  homepage 'http://code.google.com/p/ppss/'
```

Comment:
Indeed, thanks. It's a page that has mixed content, so tried it with plain http to no avail, missed to put back https.

I didn't update Google Code links to https that already use above new naming scheme, _and_ have mixed-content - even though modern browsers will redirect to https automatically anyway. Maybe it should be done.